msteams-platform/apps-in-teams-meetings/teams-live-share-capabilities.md

```diff
@@ -213,26 +215,31 @@ function onClickAddToPlaylist(video: IVideo) {
 
 ## Live Share data structures
 
-The Live Share SDK includes a set of new Live Share `SharedObject` classes, which provide stateful and stateless objects that aren't stored in the Fluid container. For example, if you want to create a laser-pointer feature into your app, such as the popular PowerPoint Live integration, you can use our `LiveEvent` or `LiveState` objects.
+The Live Share SDK includes a set of new Live Share `SharedObject` classes, which provide stateful and stateless objects. Unlike Fluid data structures, these features do not write changes to the Fluid container, enabling faster synchronization. These objects were designed from the ground up for common meeting scenarios for Teams meetings. Common scenarios include synchronizing what content the presenter is viewing, displaying metadata for each user in the meeting, or displaying a countdown timer.
```

Comment:
```suggestion
The Live Share SDK includes a set of new Live Share `SharedObject` classes, which provide stateful and stateless objects. Unlike Fluid data structures, `SharedObject` classes don’t write changes to the Fluid container, enabling faster synchronization. `SharedObject` classes were designed from the ground up for common meeting scenarios in Teams meetings. Common scenarios include synchronizing what content the presenter is viewing, displaying metadata for each user in the meeting, or displaying a countdown timer.
```

Reply:
`SharedObject` is a Fluid class and not something implemented by the Live Share SDK. We do extend it but in the context of this sentence, this is not accurate. I will update before committing.